tests/test_cli/test_push.py

```diff
@@ -91,8 +91,11 @@ def test_vendor_ok(
         copy_tree_mock,
     ):
         """Test ok for vendor's item."""
-        with mock.patch("os.path.exists", side_effect=[False, True, False]):
-            self.invoke("push", "--local", "skill", "fetchai/echo")
+
+        with mock.patch("click.core._"):
+            with mock.patch("os.path.exists", side_effect=[False, True, False]):
+                self.invoke("push", "--local", "skill", "fetchai/echo")
```

Comment:
Just a patch to fix the test failure caused by new click version

Reply:
Does that hint at an issue with the implementation?

Reply:
No, this is just a case of `click.core._` method using `os.path.exists` method which is supposed to be patched with 3 side effects which caused the lack of effects crashing the test, So I patched the method.